tests/components/config/test_config_entries.py

```diff
@@ -375,25 +384,35 @@ class TestFlow(core_ce.ConfigFlow):
 
     with patch.dict(HANDLERS, {"test": TestFlow}):
         form = await hass.config_entries.flow.async_init(
-            "test", context={"source": "hassio"}
-        )
+            "test", context={"source": "hassio"})
 
-    await ws_client.send_json({"id": 5, "type": "config_entries/flow/progress"})
+    await ws_client.send_json({
+        "id": 5,
+        "type": "config_entries/flow/progress"
+    })
     response = await ws_client.receive_json()
 
     assert response["success"]
-    assert response["result"] == [
-        {"flow_id": form["flow_id"], "handler": "test", "context": {"source": "hassio"}}
-    ]
+    assert response["result"] == [{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4752910)